router/batchrouter/asyncdestinationmanager/snowpipestreaming/snowpipestreaming.go

```diff
@@ -88,6 +88,7 @@ func New(
 	}))
 
 	m.stats.discards = statsFactory.NewTaggedStat("snowpipe_streaming_discards", stats.CountType, tags)
+	m.stats.pollingInProgress = statsFactory.NewTaggedStat("snowpipe_streaming_polling_in_progress", stats.CountType, tags)
```

Comment:
I think we can keep it simple, since the only thing that I want to measure here is how many times Poll is called and we are still waiting for few of the channels to get flushed.